src/state/transactions/types.ts

```diff
@@ -18,21 +18,23 @@ interface SerializableTransactionReceipt {
  * These values is persisted in state and if you change the value it will cause errors
  */
 export enum TransactionType {
-  APPROVAL = 0,
-  SWAP = 1,
-  DEPOSIT_LIQUIDITY_STAKING = 2,
-  WITHDRAW_LIQUIDITY_STAKING = 3,
-  CLAIM = 4,
-  VOTE = 5,
-  DELEGATE = 6,
-  WRAP = 7,
-  CREATE_V3_POOL = 8,
-  ADD_LIQUIDITY_V3_POOL = 9,
-  ADD_LIQUIDITY_V2_POOL = 10,
-  MIGRATE_LIQUIDITY_V3 = 11,
-  COLLECT_FEES = 12,
-  REMOVE_LIQUIDITY_V3 = 13,
-  SUBMIT_PROPOSAL = 14,
+  APPROVAL,
```

Comment:
Retain `APPROVAL = 0` to ensure this remains a zero-indexed enum.

Reply:
Do these enum values need to be stable across versions? Are they stored anywhere?

Reply:
the typescript compiler makes it pretty clear that it's still 0-indexed, but making it explicit can't hurt, added

i did a brief check when removing this and didn't find anything of concern. also worth noting that the values aren't actually changing